implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdLineBuilder.java

```diff
@@ -61,9 +60,10 @@
         return numberFormatter;
     });
 
-    StatsdLineBuilder(Meter.Id id, StatsdFlavor flavor, NamingConvention convention) {
+    StatsdLineBuilder(Meter.Id id, StatsdFlavor flavor, HierarchicalNameMapper nameMapper, NamingConvention convention) {
```

Comment:
Maybe overload the constructor with `HierarchicalNameMapper.DEFAULT` as the default.

Reply:
Yeah...I thought that `build` was the default task for the project, so I didn't actually run the tests before pushing. :/